roles/devscripts/tasks/build_config.yml

```
@@ -40,6 +40,16 @@
       devscripts_config_patches
     }}
 
+- name: Replace OCP version if "stable-" alias used
+  when:
+    - cifmw_devscripts_config.openshift_version.startswith("stable-")
+  vars:
+    ocp_release_txt: "{{ lookup('ansible.builtin.url', 'https://mirror.openshift.com/pub/openshift-v4/clients/ocp/' ~ cifmw_devscripts_config.openshift_version ~ '/release.txt', split_lines=False) }}"
```

Could you please prefix these internal vars with _ like so? they are only used in this one task so we should denote them as internal and we don't have to document them.
```suggestion
    _ocp_release_txt: "{{ lookup('ansible.builtin.url', 'https://mirror.openshift.com/pub/openshift-v4/clients/ocp/' ~ cifmw_devscripts_config.openshift_version ~ '/release.txt', split_lines=False) }}"
```